components/micro-gateway-core/src/main/ballerina/gateway/filters/subscription_filter.bal

```diff
@@ -82,6 +83,25 @@ public type SubscriptionFilter object {
                                     return true;
                                 }
                             }
+                            }
+                            else
+                            {
```

Comment:
fix formatting issue

Reply:
this code has revamped and formatted with 4d34de4